esmvalcore/preprocessor/_time.py

```diff
@@ -8,16 +8,18 @@
 import logging
 from warnings import filterwarnings
 
+import cf_units
 import dask.array as da
 import iris
 import iris.coord_categorisation
 import iris.cube
 import iris.exceptions
 import iris.util
+import isodate
```

Comment:
Please add the new dependency in the [relevant files](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/contributing.html#dependencies)

Reply:
Added in #1133